packages/karma-esm/middleware.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs');
+const path = require('path');
+const esm = require('./src/karma-esm');
```

Comment:
consider using `karmaEsm` for variable name, as in the future you might want to use `esm` package for interop

Reply:
Good idea, made the changes.